slimta/diskstorage/__init__.py

```diff
@@ -254,8 +264,11 @@ def set_recipients_delivered(self, id, rcpt_indexes):
 
     def load(self):
         for id in self.ops.get_ids():
-            meta = self.ops.read_meta(id)
-            yield (meta['timestamp'], id)
+            try:
+                meta = self.ops.read_meta(id)
+                yield (meta['timestamp'], id)
+            except OSError, e:
+                logging.logging.getLogger(__name__).exception("Cannot load id %r from queue" % id)
```

Comment:
I like to keep logs coming out of slimta and its plugins structured and consistent. Please change this to:

```python
except OSError:
    logging.log_exception(__name__, queue_id=id)
```

That should automatically populate the error line with applicable information.